WordPress/UITests/Tests/ReaderTests.swift

```diff
@@ -7,8 +7,8 @@ class ReaderTests: XCTestCase {
     override func setUpWithError() throws {
         setUpTestSuite()
 
-        _ = try LoginFlow.login(siteUrl: WPUITestCredentials.testWPcomSiteAddress, email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
-        readerScreen = try EditorFlow
```

Comment:
With the test no longer using the stored `readerScreen`, we should be able to remove the property from the class, too. See line 5 of this file.